UnityProject/Assets/Scripts/Objects/Pipes/Devices/TemperatureGate.cs

```diff
@@ -95,5 +97,42 @@ public override void TickUpdate()
 				pipeData.mixAndVolume.EqualiseWithOutputs(pipeData.ConnectedPipes);
 			}
 		}
+
+
+		public string Examine(Vector3 worldPos = default)
+		{
+			return $"The gate has been set to let gas flow when the input gas temp is {(isInverted ? "higher" : "lower")} than set threshold.";
+		}
+
+		public string HoverTip()
+		{
+			return null;
+		}
+
+		public string CustomTitle()
+		{
+			return null;
+		}
+
+		public Sprite CustomIcon()
+		{
+			return null;
+		}
+
+		public List<Sprite> IconIndicators()
+		{
+			return null;
+		}
+
+		public List<TextColor> InteractionsStrings()
+		{
+			var list = new List<TextColor>
+			{
+				new() { Color = Color.green, Text = "Left Click: Toggle Power." },
+				new() { Color = Color.green, Text = $"Left Click with screwdriver: {(isInverted ? "Reset" : "Invert" )} temperature sensor." },
+				new() { Color = Color.green, Text = "Alt Click: Open GUI." }
```

Comment:
Fun fact: you can make some of these context aware.
For example, you can make the "Open GUI" tip only show up if they're close to it. Not asking you to do this right now, but its something to keep in mind in the future.